api/v1_projects.inc

```diff
@@ -880,6 +880,32 @@ function api_v1_project_page(string $method, array $data, array $query_params)
     }
 }
 
+function api_v1_project_page_report_bad(string $method, array $data, array $query_params): void
```

Comment:
```suggestion
/** @param array<string, string|string[]> $query_params */
function api_v1_project_page_report_bad(string $method, array $data, array $query_params): void
```

To get ahead of @jchaffraix's PR that this will miss.